custom-exporter/exporter.py

```diff
@@ -63,7 +64,8 @@ def fetch_data(self, url):
             print(f"Error fetching data from {url}: {e}")
             return None
         
-    def fetch_events(self, node_address):
+    def fetch_events(self, node_address, batch_size=100000):
```

Comment:
I will suggest using batch_size as 10000. Although most rpc providers limits event results instead of block range like Infura, but just in case. This will only increase the number of queries for the first run. Also for reference https://docs.chainstack.com/docs/understanding-eth-getlogs-limitations

Reply:
btw I've also seen a more secure way handling block range issue like this https://ethereum.stackexchange.com/questions/79672/how-do-i-get-all-logs-for-all-time-for-a-given-contract-using-web3/79675#79675